pkg/platform/interface.go

```diff
@@ -3,11 +3,8 @@ package platform
 import "go.keploy.io/server/pkg/models"
 
 type TestCaseDB interface {
-	WriteTestcase(path string, tc *models.TestCase) error
-	WriteMock(path string, tc *models.Mock) error
-
-	NewSessionIndex(path string) (string, error)
-	ReadSessionIndices(path string) ([]string, error)
```

Comment:
I think we should have ReadSessionIndices and NewSessionIndex in the interface because implementation would be different for each type of data store.

Reply:
I have removed them because I didn't see them depending on any other elements of the struct other than path which is being passed in the function . I think sessionIndices are more independent from writing mocks and tests although it can be used to be a part of path .

Reply:
I have also made a recent change 

Removed config.yaml and added them in mocks.yaml.

please review this change too